Prototyper/Classes/Feedback/FeedbackView.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  FeedbackView.swift
+//  KeychainSwift
+//
+//  Created by Raymond Pinto on 20.12.19.
+//
+
+import SwiftUI
+
+struct FeedbackView: View {
+    @EnvironmentObject var model: Model
+    @State var descriptionText: String = ""
+    @State var showSheet: Bool = false
+    @State var showScreenshot: Bool = true
+    @State var activeSheet: ActiveSheet = .loginSheet
+    @State var showSendFeedbackView: Bool = false
+    @State var feedback: Feedback?
+    
+    enum ActiveSheet {
+        case loginSheet, MarkupSheet
+    }
+    
+    var body: some View {
+        NavigationView {
+            VStack {
+                HStack (alignment: .top) {
+                    if showScreenshot {
+                        ZStack (alignment: .topTrailing) {
+                            ZStack {
+                                Image(uiImage: model.screenshot)
+                                    .resizable()
+                                    .border(Color(white: 0.75))
+                                    .scaledToFit()
+                                Image(systemName: "hand.draw.fill")
```

Comment:
I would suggest to add some sort of circle that is semitransparent as a background of the Image so it is clearer differentiable from the background. The circle might also have some sort of shadow, that might look nice :)

Reply:
I found a better image: "pencil.circle.fill". If you think the previous one was better I could just overlap the "hand.draw" over "circle.fill".